### PR TITLE
Add Instructions for running CFL in Prow

### PR DESCRIPTION
**Pull request by mpherman2**
@jonathanmetzman PTAL

### CODE REVIEW

File: docs/running-clusterfuzzlite/prow.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+Then you must create a service account with access to both `<your-cloud-bucket>` and the Prow Results Bucket
+To do this
+  1) Create a Cloud Service account in your Project
+
+    gcloud beta iam service-accounts create `<SA_NAME>` --project=`<PROJECT_ID>` --description=`<SA_DESCRIPTION>` --display-name=`<SA_DISPLAY_NAME>`
+
+  2) Give the Cloud Service account access to BOTH the k8s Prow Results Bucket AND `<your-cloud-bucket>` using the command
+
+    gsutil iam ch "serviceAccount:${SA}:roles/storage.objectAdmin" "${BUCKET}"
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/prow.md:34]: Should `$SA` be SA_NAME or something else?

File: docs/running-clusterfuzzlite/prow.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+To enable batch fuzzing, add the following to your prow job config for periodic jobs.
+
+{% raw %}
+```yaml
+- name: <YOUR-PROWJOB-NAME>
+  labels:
+    preset-dind-enabled: "true"
+  decorate: true
+  extra_refs:
+  - org: <ORG>
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/prow.md:105]: Can you add an explanation for <ORG> <REPO> and <BRANCH> please?

File: docs/running-clusterfuzzlite/prow.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    repo: <REPO>
+    base_ref: <BRANCH>
+  interval: 1h # THIS CAN BE A CRON AS WELL
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/prow.md:108]: Isn't this going to run hourly? How can we make it run daily?

File: docs/running-clusterfuzzlite/prow.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+Artifacts produced by ClusterFuzzLite are stored in `<your-cloud-bucket>`.
+The bucket has the following layout.
+1. Coverage report: <your-cloud-bucket>/coverage/latest/report/linux/index.html
+1. Builds: <your-cloud-bucket>/build/<sanitizer>-<commit>/
+1. Corpora: <your-cloud-bucket>/corpus/<fuzz-target>/
+1. Crashes: <your-cloud-bucket>/crashes/<fuzz-target>/<sanitizer>/<crash-file> and <your-cloud-bucket>/crashes/<fuzz-target>/<sanitizer>/<crash-file>.summary (output from crash).
+
+## Viewing Crashes
+
+You can view your crashes on Deck by clicking the Artifacts button on the top of the page for your prowjob run. Crashes will be located in
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/prow.md:237]: A good todo for later would be to get a screenshot of this (need to do this for actions and GCB too).

File: docs/running-clusterfuzzlite/prow.md (hunk 1 above)
Comment (mpherman2) [reply docs/running-clusterfuzzlite/prow.md:34]: Yea. <SA_NAME> fits what we have been doing in the rest of the docs

File: docs/running-clusterfuzzlite/prow.md (hunk 3 above)
Comment (mpherman2) [reply docs/running-clusterfuzzlite/prow.md:108]: interval: 24h

File: docs/running-clusterfuzzlite/prow.md (hunk 4 above)
Comment (mpherman2) [reply docs/running-clusterfuzzlite/prow.md:237]: You can leave me a todo and I will add it in a new PR

**Pull request merged**